tt1-blocks/style.css

```diff
@@ -45,36 +45,17 @@ body {
 	padding: 0 var(--wp--custom--spacing--horizontal);
 }
 
-.wp-site-blocks > *:not(.wp-block-post-content),
-.wp-site-blocks .wp-block-post-content > * {
-	max-width: var(--wp--custom--responsive--aligndefault-width);
+.wp-site-blocks * {
 	margin-left: auto;
 	margin-right: auto;
 }
 
-.wp-site-blocks .alignwide {
-	width: var(--wp--custom--responsive--alignwide-width);
-	max-width: 100%;
-	margin-left: auto;
-	margin-right: auto;
-}
-
-.wp-site-blocks .alignfull {
-	transform: translateX(calc(0px - var(--wp--custom--spacing--horizontal)));
-	width: calc(100% + (2 * var(--wp--custom--spacing--horizontal)));
-	max-width: calc(100% + (2 * var(--wp--custom--spacing--horizontal)));
-	margin-left: 0;
-	margin-right: 0;
-	box-sizing: content-box;
-}
-
-.wp-site-blocks .wp-block-template-part.alignfull {
-	width: 100%;
-	max-width: 100%;
+.wp-site-blocks *:not(.wp-block-post-content):not(.alignfull):not(.alignwide) {
```

Comment:
```suggestion
.wp-site-blocks *:not(.wp-block-post-content):not(.alignfull):not(.alignwide):not([class$="__inner-container"]) {
 {
```

Exclusing inner containers (Group, Cover, etc) from this rule seems to solve the alignment issues noted [here](https://github.com/WordPress/theme-experiments/pull/148#issuecomment-748175575). 

Before|After
---|---
![Screen Shot 2020-12-18 at 2 32 31 PM](https://user-images.githubusercontent.com/1202812/102654020-dfe3f080-413d-11eb-8610-2018f2dd31c8.png)|![Screen Shot 2020-12-18 at 2 32 09 PM](https://user-images.githubusercontent.com/1202812/102653993-d195d480-413d-11eb-943c-dc76ae7eb840.png)